src/components/select/select.scss

```
@@ -59,6 +59,24 @@ limel-notched-outline:not([invalid]:not([invalid='false'])) {
     margin-right: functions.pxToRem(8);
     margin-left: functions.pxToRem(-8);
     flex-shrink: 0;
+
+    &--inline {
+        margin-left: 0;
+        margin-right: functions.pxToRem(4);
+    }
+}
+
+.limel-select__selected-option-item-wrapper {
```

Stylelint (17.2.0)

[error] Expected class selector ".limel-select__selected-option-item-wrapper" to be kebab-case (selector-class-pattern)

(selector-class-pattern)